src/app/view/meter-read/add-bulk-reads/add-bulk-reads.component.scss

```diff
@@ -0,0 +1,197 @@
+.body-bg {
+    padding: 35px 30px 35px 30px;
+    background: #f2f2f2;
+  
+    .mat-mdc-card {
+      box-shadow: 3px 0px 33px rgba(0, 0, 0, 0.1) !important;
+      border: none !important;
+    }
+  }
+  
+  p {
+    font-family: Lato;
+  }
+  
+  .output {
```

Comment:
I think we don't need all these class names. I would prefer to not introduce too many classnames every time we make a change